module-system/module-implementations/sov-evm/src/evm/transaction.rs

```diff
@@ -25,62 +29,54 @@ impl Default for BlockEnv {
     }
 }
 
+/// Rlp encoded evm transaction.
 #[cfg_attr(
     feature = "native",
     derive(serde::Serialize),
     derive(serde::Deserialize),
     derive(schemars::JsonSchema)
 )]
 #[derive(borsh::BorshDeserialize, borsh::BorshSerialize, Debug, PartialEq, Clone)]
-pub struct AccessListItem {
-    pub address: EthAddress,
-    pub storage_keys: Vec<Bytes32>,
+pub struct RawEvmTransaction {
+    /// Rlp data.
+    pub tx: Vec<u8>,
```

Comment:
lets just call it `rlp` to avoid confussion

Reply:
done in `4d9f1f0bc9`